druidfi/spell
# Drupal 8.8

## Changes
 
- new core 8.8
- scaffolding plugin from Drupal
- updated docs
- removed amazee.io related setup

## How to test

You need stonehenge up & running and PHP & Composer on your host.

- `composer create-project druidfi/spell:dev-drupal-8.8 mytestsite --no-interaction`
- `cd mytestsite`
- `make new DRUPAL_PROFILE=demo_umami`

You should get `drush uli` link to created site.
